app/models/ontology_member/symbol.rb

```diff
@@ -11,10 +11,14 @@ class Symbol < ActiveRecord::Base
     has_and_belongs_to_many :sentences
     has_and_belongs_to_many :oops_responses
 
+    attr_accessible :locid
     attr_accessible :label, :comment
 
     scope :kind, ->(kind) { where kind:  kind }
 
+    def self.find_with_locid(locid, _iri=nil)
```

Comment:
Style/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.